src/base/bittorrent/session.cpp

```diff
@@ -4524,14 +4527,15 @@ void Session::startUpTorrents()
                 {
                     recoveredCategories.insert(category);
                     isCategoryRecovered = true;
-                    LogMsg(tr("Inconsistent data is detected. Category '%1' is assigned to some torrent(s) but it doesn't exist in the configuration file."
-                              " Its settings will be reset to default.").arg(category), Log::WARNING);
+                    LogMsg(tr("Detected inconsistent data: torrent contains category that doesn't exist in the configuration file."
+                              " Category settings of the affected torrent will be reset."
```

Comment:
It is incorrect completely.
Missing category will be recovered with default settings (i.e. save/download paths).